external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -61,15 +61,42 @@
 
     <Copy Condition="'$(UAPToolsFolder)'!=''"
           SourceFiles="@(RunnerFolderContents)" 
-          DestinationFolder="$(NETCoreAppTestRootPath)\Runner\%(RecursiveDir)"
+          DestinationFolder="$(TestHostRootPath)\Runner\%(RecursiveDir)"
           SkipUnchangedFiles="true"/>
 
     <Copy Condition="'$(UAPToolsFolder)'!=''"
           SourceFiles="@(LauncherFolderContents)" 
-          DestinationFolder="$(NETCoreAppTestRootPath)\Launcher\%(RecursiveDir)"
+          DestinationFolder="$(TestHostRootPath)\Launcher\%(RecursiveDir)"
           SkipUnchangedFiles="true"/>
   </Target>
 
+  <Target Name="VerifyMatchingProjectNVersions"
+          BeforeTargets="Build">
+    <Error Condition="'$(ProjectNTfsExpectedPrerelease)' != '$(ProjectNTfsTestILCExpectedPrerelease)'" Text="The versions of ProjectN Targeting pack and TestILC have gone out of sync. Please make sure both of the prerelease versions on dependencies.props match." />
+  </Target>
+
+  <Target Name="CopyTestILCToolsToTestHost"
+          AfterTargets="RestorePackages"
+          Condition="'$(TargetGroup)' == 'uapaot'">
+    <PropertyGroup>
+      <TestILCToolsPackageName>TestILCNugetPackageForCoreFX</TestILCToolsPackageName>
+      <TestILCFolder Condition="'$(TestILCFolder)' == ''">$(PackagesDir)$(TestILCToolsPackageName)\1.0.0-$(ProjectNTfsTestILCExpectedPrerelease)\TestILC</TestILCFolder>
```

Comment:
It would be slightly better to replace `1.0.0-$(ProjectNTfsTestILCExpectedPrerelease)` with a new property `$(TestILCNugetPackageForCoreFXPackageVersion)` to avoid embedding the `1.0.0-` here. Then you have to add a `TestILCNugetPackageForCoreFXPackageVersion` property and upgrader in dependencies.props, though. (See `NETStandardPackageVersion`.)